### ISSUE TITLE
Remove S4 from the table

### ISSUE THREAD

**Issue by lennart-huesing** [CombiRWTH/medical-staff-assessment#157]
This is not needed frontend wise

**Comment by lennart-huesing** [comment]
Remove s4 from the returned information from the endpoint

**Comment by lennart-huesing** [comment]
Also remove a1, s1

**Comment by aalhousseini** [comment]
do you mean from the get_all_stations() ?

**Comment by lennart-huesing** [comment]
No. When you open the classifction view for a patient, a table of questions is displayed.
In that table there  are the columns A1, S1, S4. This information is pulled from the backend and should be modified, so the said columns are not returned.
